### PR TITLE
Fix header color to be consistent

### PR DESCRIPTION
**Pull request by thaJeztah**
See individual commits for details

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya @StefanScherer ptal

File: _includes/body-landing.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<body ng-app="Docker" ng-controller="DockerController" id="landing">
+<body ng-app="Docker" ng-controller="DockerController" id="landing" class="landing">
```
Comment (thaJeztah) [inline _includes/body-landing.html:1]: I think the id, and the `ng-` attributes can be removed now, but will look separately at those.

**Comment by StefanScherer** [comment]
Agreed, I also could see the same color in the header. LGTM

I also tried on mobile, and I have some findings, but unrelated to this PR, I saw that on docs.docker.com as well.

docs-mobile

**Comment by thaJeztah** [comment]
Oh, yes, I think I saw the "black" color for the header somewhere, and looked like it was possibly done on purpose, but admit I didn't check on my phone how it looked

**Comment by usha-mandya** [comment]
Thanks Stefan. I checked the light and dark modes but didn't check the responsive design on the Netlify preview.

**Pull request merged**